### PR TITLE
Fix qc round 5

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
SES-159-qa-bug-findings-v-30-r

# Actions
Change name button place holder and add text button
Add correct gap and button text in mobile

### CODE REVIEW

File: src/stories/components/cu-activity-table/cu-activity-table.placeholder.tsx [hunk 1]
```diff
@@ -37,9 +39,12 @@ export const ActivityPlaceholder = (props: { hasFilter: boolean }) => {
             buttonType={ButtonType.Secondary}
           />
         )}
-        <a href={SUBMIT_EXPENSES_URL} target="_blank">
-          <CustomButton style={{ minWidth: '250px' }} label="Submit Expenses Now" buttonType={ButtonType.Primary} />
-        </a>
+        <CustomButton
```
Comment (YasielCabrera) [inline src/stories/components/cu-activity-table/cu-activity-table.placeholder.tsx:42]: can this be a `Link` (from nextjs) to navigate instead a button with a `onClick` event?

File: src/stories/components/reset-button/reset-button.tsx [hunk 2]
```diff
@@ -6,27 +6,44 @@ import { Close } from '../svg/close';
 interface Props {
   onClick?: () => void;
   disabled?: boolean;
+  label?: string;
+  isWithIcon?: boolean;
+  labelMobile?: string;
 }
 
-export default (props: Props) => {
+export default ({ onClick, disabled, label = 'Reset Filters', isWithIcon = true, labelMobile }: Props) => {
```
Comment (YasielCabrera) [inline src/stories/components/reset-button/reset-button.tsx:14]: we can use `hasIcon` here as it is less verbose and still refers to a boolean according to the conventions

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/components/reset-button/reset-button.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/components/reset-button/reset-button.tsx:14]: ok, I will make the change

File: src/stories/components/cu-activity-table/cu-activity-table.placeholder.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply src/stories/components/cu-activity-table/cu-activity-table.placeholder.tsx:42]: You are right, in next refactoring, we will extend the component to have both functions

**Pull request merged**